libs/widget-lib/src/types.ts

```diff
@@ -1,4 +1,5 @@
 import type { SupportedChainId } from '@cowprotocol/cow-sdk'
+export type { SupportedChainId } from '@cowprotocol/cow-sdk'
```

Comment:
Why?

Reply:
Because is a type, it shouldn't harm the size of anything, and otherwise you would depend on '@cowprotocol/cow-sdk' to get it.

Think that the widget lib is needed by projects that don't depend in the SDK, but we ask them for the `chainId`, but we don't export its type. 

It would be great to let them import this type without adding the `SDK`

Does it make sense?

Reply:
Yes, the source of truth comes from CoW SDK.
This is just a small thing to make it more convenient for clients of the widget to import this type.

I'm fine changing this, but I want to understand exactly what you mean, cause I think I don't get what is the problem. You suggest that people import from the SDK instead. If so, i can do that, is just one step away for people, as its so relevant field part of the params signature Im re-exporting

Reply:
No issues here, this is fine.
I'm just highlighting that, even though people won't be importing from cow-sdk directly, cow-sdk will still be a dependency for the widget.

Reply:
yes, yes, but what i wanted is to make it a transitive dependency that won't be adding any weight in the bundle, and also would make it easier for anyone to get the SupportedNetworks from the exported artifacts

Reply:
> won't be adding any weight in the bundle

Is that a thing? Sorry my ignorance, I thought that anything that's depended on would be required in order to build the lib.

Reply:
My point is that as this was as type, is not in the runtime, so no weight or dependencies i believe. Of course there's a dependency on build time, there is even if I don't re-export, right?